dashboard/test/factories/factories.rb

```diff
@@ -1680,4 +1680,14 @@
     data_synced {{db_Opt_In: 'No'}}
     data_synced_at {Time.now.utc}
   end
+
+  factory :lti_integration do
+    platform_id {"1a2b3c5b"}
+    issuer {"http://some.lms.com"}
+    client_id {"12345678"}
+    platform_name {"canavs"}
+    auth_redirect_url {"http://some.lms.com/lti/authenticate"}
+    jwks_url {"http://some.lms.com/jwks"}
+    access_token_url {"http://some.lms.com/access_token"}
+  end
```

Comment:
Do we tend to use hardcoded values in factories, or do we ever prefer randomization? I honestly don't know. I'm a big fan of random so we discover edge cases earlier.

Reply:
Interesting, I'll take a look at some other factories for inspiration.

Reply:
It seems like a lot of values in factories are hardcoded. However I've changed these to just be string values named as their key. If there is a preferred method, I'm happy to change this.